### PR TITLE
MBS-8132 Support no report-viewer

### PR DESCRIPTION
**Pull request by sergioboi**

### CODE REVIEW

File: subprojects/gradle/instrumentation-tests/build.gradle.kts [hunk 1]
```diff
@@ -30,6 +30,7 @@ dependencies {
     implementation(project(":common:sentry"))
     implementation(project(":common:logger"))
     implementation(project(":gradle:upload-cd-build-result"))
+    implementation(project(":gradle:worker-worker"))
```
Comment (eugene-krivobokov) [inline subprojects/gradle/instrumentation-tests/build.gradle.kts]: Maybe name it just `worker`?

File: subprojects/gradle/worker-worker/src/main/kotlin/com/avito/gradle/worker/WorkerExt.kt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import org.gradle.workers.WorkParameters
+import org.gradle.workers.WorkerExecutor
+
+fun WorkerExecutor.inMemoryWork(work: () -> Unit) {
+    noIsolation().submit(NoSerializableWork::class.java) { params ->
+        params.state = NoSerializationWorkerParams.StateHolder(work)
+    }
+}
+
+internal abstract class NoSerializableWork<T> : WorkAction<NoSerializationWorkerParams> {
```
Comment (eugene-krivobokov) [inline subprojects/gradle/worker-worker/src/main/kotlin/com/avito/gradle/worker/WorkerExt.kt]: `NoSerializableWork` -> `NonSerializableWork`

**Review by dsvoronin — APPROVED** [review]

**Comment by eugene-krivobokov** [comment]
Have you checked local run and fullCheck?

**Review by eugene-krivobokov — APPROVED** [review]

**Comment by sergioboi** [comment]
That weren't all my changes.
This weekend changes i will push in another branch

**Pull request merged**